converter.go

```diff
@@ -28,6 +28,7 @@ import (
 
 const format = "2006-01-02 15:04:05.999999999"
 const numberDefaultPrecision = 38
+const jsonFormatStr = "json"
```

Comment:
@sfc-gh-dszmolka  can you please explain how the format string is being used by the ExecBindQuery, or what is the purpose of this field?

Reply:
Great question! This field is extremely important, when dealing with structured objects/arrays/maps. If a field is structured, we need to inform snowflake backend, that it should parse this value and treat it as structured type instead of plain string.